recipe_scrapers/greatbritishchefs.py

```diff
@@ -28,15 +31,15 @@ def total_time(self):
                 total_time = tt2
         return total_time
 
-    def yields(self):
+    def yields(self) -> Optional[str]:
         recipe_yield = self.soup.find("span", {"class": "RecipeAttributes__Serves"})
         if recipe_yield:
-            yields = normalize_string(
+            recipe_yield = normalize_string(
                 recipe_yield.find("span", {"class": "header-attribute-text"}).get_text()
             )
-            return yields
+        return recipe_yield
```

Comment:
Taking a mental note to look at some of this logic again during #329.

Reply:
Please do! We might need to adjust more of the schema logic for the defaulting.